builds/android/res/values/strings.xml

```diff
@@ -80,6 +80,7 @@ Please tell us in detail what went wrong.\n
 	<string name="add_game_folder">Add a game folder</string>
 	<string name="enable_vibration">Enable vibrations</string>
 	<string name="vibrate_when_sliding_direction">Vibrate when sliding to another direction</string>
+	<string name="disable_sounds">Disable sounds</string>
```

Comment:
Because it's a check box it should be checked by default and have the label "Enable audio" (Disabling by checking is confusing)